client/src/components/create-course-view/CreateCourseForm.js

```diff
@@ -0,0 +1,98 @@
+// SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import TextFieldBox from './TextFieldBox';
+import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+
+const codeData = {
+  fieldId: 'courseCode',
+  fieldLabel: 'Course Code',
+  fieldHelperText: 'Give the code that the new course is going to have.'
+};
+
+const nameData = {
+  fieldId: 'courseName',
+  fieldLabel: 'Course Name',
+  fieldHelperText: 'Give the name of the course the new course is going to have.'
+};
+
+const organizerData = {
+  fieldId: 'organizer',
+  fieldLabel: 'Organizer',
+  fieldHelperText: 'Give the organizer of the new course.'
+};
+
+/*const teachersData = {
+  fieldId: 'teachers',
+  fieldLabel: 'Responsibe Teachers',
+  fieldHelperText: 'Give the emails of the responsible teachers of the new course.'
+};*/
+
+const CreateCourseForm = ({ addCourse }) => {
+
+  const [courseCode, setCode] = useState('');
+  const [name, setName] = useState('');
+  const [department, setOrganizer] = useState('');
+  //const [teacher, setTeacher] = useState(''); 
+
+  const id = 100;
+  const minCredits = 5;
+  const maxCredits = 5;
+  const evaluationInformation = 'General scale, 0-5';
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    try {
+      const courseObject = ({
+        id,
+        courseCode,
+        minCredits,
+        maxCredits,
+        department: {
+          fi: '',
+          sv: '',
+          en: department,
+        },
+        name: {
+          fi: '',
+          sv: '',
+          en: name,
+        },
+        evaluationInformation: {
+          fi: '',
+          sv: '',
+          en: evaluationInformation,
+        }
+        //teacher
+      });
+      addCourse(courseObject);
+    } catch (exception) {
+      console.log(exception);
+    }
+  };
+
+  return(
+    <Container maxWidth="sm" sx={{ textAlign: 'right' }}>
+      <form onSubmit={handleSubmit}>
+        <TextFieldBox fieldData={codeData} setFunction={setCode}/>
+        <TextFieldBox fieldData={nameData} setFunction={setName}/>
+        <TextFieldBox fieldData={organizerData} setFunction={setOrganizer}/>
```

Comment:
There will be a finite number of organisers but there might be multiple responsible teachers. For this the idea is to add the possibility to add multiple inputs for the TextField (or a multiline input). But this will be looked into when the attributes for a course and course instance are fixed according to the issue #70.